src/script/main/app.ts

```diff
@@ -120,12 +121,18 @@ import {ConnectionService} from '../connection/ConnectionService';
 import {TeamService} from '../team/TeamService';
 import {SearchService} from '../search/SearchService';
 import {CryptographyService} from '../cryptography/CryptographyService';
-import {AccessTokenError} from '../error/AccessTokenError';
+import {AccessTokenError, ACCESS_TOKEN_ERROR_TYPE} from '../error/AccessTokenError';
 import {ClientError} from '../error/ClientError';
 import {AuthError} from '../error/AuthError';
 import {AssetRepository} from '../assets/AssetRepository';
+import {SQLeetEngine} from '@wireapp/store-engine-sqleet';
```

Comment:
Please move this import to the top, next to the other external imports. Also please use `import type`.

Reply:
done in https://github.com/wireapp/wire-webapp/pull/9511/commits/f89353a3a5793eaa4bf6a95c1cd629ec63828ee1